blazor/getting-started/maui-blazor-app.md

```diff
@@ -165,6 +165,16 @@ Before running the sample, make sure the mode is `Windows Machine`.
 
 > Download demo from [GitHub](https://github.com/SyncfusionExamples/MAUI-Blazor-App-using-Syncfusion-Blazor-Components)
 
+## How to use images in .NET MAUI Blazor Application
+
+Add the images in the images in the `wwwroot` folder of the application and refer it using `img` tag in Blazor App. 
+
+In the below code images are added under `images` folder in `wwwroot` folder.
```

Comment:
provide code examples also.

Reply:
When we add code snippets, in preview it is directly executing the tag sir, instead of displaying the code. Hence, added the images for code snippets

![image](https://user-images.githubusercontent.com/93309503/196095871-1597ee33-1cc9-4b67-a652-7a1a51a28b1e.png)

![image](https://user-images.githubusercontent.com/93309503/196095889-88d64395-900d-4214-ade1-48b8b3084722.png)

Reply:
Added code-snippets and removed the image sir.